1.caching-with-weakmap/index.js

```diff
@@ -2,6 +2,19 @@
 
 function processData(salaries) {
   // start coding here
+  let weakMap = new WeakMap();
```

Comment:
```suggestion
  let weakMap = new WeakMap();
```
Kode ini seharusnya diletakkan di luar function, agar weakMap tidak tereset setiap kali memanggil function